sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventParticipantEditForm.java

```diff
@@ -84,6 +86,11 @@ protected void addFields() {
 			new PersonEditForm(PersonContext.EVENT_PARTICIPANT, event.getDisease(), event.getDiseaseDetails(), null, isPseudonymized);
 		pef.setWidth(100, Unit.PERCENTAGE);
 		pef.setImmediate(true);
+
+		CountryReferenceDto countryDto = event.getEventLocation().getCountry();
+		pef.getAddressForm().getField(LocationDto.COUNTRY).setEnabled(false);
+		pef.getAddressForm()
```

Comment:
What needs to be set to required here is NOT the person's address, but the responsible jurisdiction fields of the event participant